apps/site/assets/css/_stop-page.scss

```diff
@@ -311,6 +311,15 @@
   }
 }
 
+.stop-page__header--fontsize {
```

Comment:
If we are going to use header tags to organize the page, I'd rather styling not be tied to them so then we don't need to make classes to overwrite existing behavior.  We'd be able to make reusable styles and apply them.
Ideally this could all be done using bootstrap 5 and there would be no need for these classes, but we aren't there yet.